### PR TITLE
New default map

### PR DESCRIPTION
**Pull request by thorvaldj**
An attempt in resolving #63. Robot and asset reference points are now equal. Resulting in no conversion between the coordinate systems. The reference points are set to be the same z=0 plane, perhaps giving an easier visualisation for tests?

### CODE REVIEW

**Comment by aestene** [comment]
Looks like a good solution! Using `z=0` as reference is a good idea.

Note that the transformation running in the background will cause some numerical computation effects on the results. Although this is of little consequence, it may be we should implement the possibility of toggling the transformation on and off in the future rather than performing the unity transformation this way. However, that is an issue for tomorrow.
image

Great contribution!

File: src/isar/config/maps/default_map.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          "x": 100.0,
+          "y": 50.0,
+          "z": 0.0,
+          "frame": "asset"
+        }
+      ],
+      "frame": "asset"
+    }
+  }
+
```
Comment (aestene) [inline src/isar/config/maps/default_map.json:50]: Add newline at end of file.

**Review by aestene — CHANGES_REQUESTED** [review]
One nitpicky comment, otherwise it looks good. Let me know when you want to move the PR from draft so it may be merged.

File: src/isar/config/maps/default_map.json (hunk 1 above)
Comment (aestene) [reply src/isar/config/maps/default_map.json:50]: Or, seems there is a newline but there are some empty spaces there.

Comment (thorvaldj) [reply src/isar/config/maps/default_map.json]: Ah. True. Force pushed a fix now.

**Comment by thorvaldj** [comment]
> Note that the transformation running in the background will cause some numerical computation effects on the results. Although this is of little consequence, it may be we should implement the possibility of toggling the transformation on and off in the future rather than performing the unity transformation this way. However, that is an issue for tomorrow.

Yes, if I understand this correctly. You could perhaps toggle the transformation on/off on whether the initial reference points are equal or not.

**Review by aestene — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**